.codegen/certificates/cert-manager/deployment.yaml

```diff
@@ -136,6 +136,7 @@ spec:
             - --v=2
             - --cluster-resource-namespace=$(POD_NAMESPACE)
             - --leader-election-namespace=$(POD_NAMESPACE)
+            - --issuer-ambient-credentials
```

Comment:
This is required for cert-manager to work with `Issuer` types and not just `ClusterIssuer`